### PR TITLE
css-box-model - Cindy Le

### PR DESCRIPTION
**Pull request by cindy-le91**
Screen Shot 2023-03-28 at 9 57 55 PM

### CODE REVIEW

File: css-box-model/styles.css [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  font-size: 24px;
+  background-color: lightgray;
+
+  /* your code here 👇 */
+  margin-left: 12px;
+  margin-right: 12px;
+  padding: 6px 16px;
+  border-radius: 6px;
+
+  /* margin-left: 12px;
```
Comment (robgmerrill) [inline css-box-model/styles.css:11]: Please remove commented out code. 
Code comments are ok, such as line 5. But commented out code should always be removed :)

**Review by robgmerrill — CHANGES_REQUESTED** [review]
Great start! Left a commented about removing commented out code.

File: css-box-model/styles.css (hunk 1 above)
Comment (cindy-le91) [reply css-box-model/styles.css]: Screen Shot 2023-03-29 at 8 41 37 AM

**Review by robgmerrill — APPROVED** [review]
👍

**Pull request merged**